lib/CustomFields/pages/ViewCustomFieldsRecord/ViewCustomFieldsRecord.js

```diff
@@ -59,6 +61,8 @@ const propTypes = {
 
 const defaultProps = {
   columnCount: 4,
+  customFieldsLabel: 'stripes-smart-components.customFields',
```

Comment:
Usually in Folio labels that are passed as props are either `<FormattedMessage` components or already formatted strings. Passing and id of a translation might be confusing to users of component.
Please change default value to `<FormattedMessage id="stripes-smart-components.customFields" />` and simply render this prop

Reply:
@BogdanDenis I'm not sure that it will be working, because `<FormattedMessage>` is the react component and we cannot use it in `defaultProps`, outside of the component

Reply:
I think that better to rename this prop in `customFieldLabelId`. Does it make sense?

Reply:
@Vladyslav-Velytskyi there's an example of `<FormattedMessage>` in default props - `<InfoPopover>` in `stripes-components`